source/index.rst

```diff
@@ -0,0 +1,114 @@
+Welcome to Darker's Documentation!
+===================================
+
+|build-badge| |license-badge| |pypi-badge| |downloads-badge| |black-badge| |changelog-badge|
+
+.. |build-badge| image:: https://github.com/akaihola/darker/actions/workflows/python-package.yml/badge.svg
+   :alt: master branch build status
+   :target: https://github.com/akaihola/darker/actions/workflows/python-package.yml?query=branch%3Amaster
+.. |license-badge| image:: https://img.shields.io/badge/License-BSD%203--Clause-blue.svg
+   :alt: BSD 3 Clause license
+   :target: https://github.com/akaihola/darker/blob/master/LICENSE.rst
+.. |pypi-badge| image:: https://img.shields.io/pypi/v/darker
+   :alt: Latest release on PyPI
+   :target: https://pypi.org/project/darker/
+.. |downloads-badge| image:: https://pepy.tech/badge/darker
+   :alt: Number of downloads
+   :target: https://pepy.tech/project/darker
+.. |black-badge| image:: https://img.shields.io/badge/code%20style-black-000000.svg
+   :alt: Source code formatted using Black
+   :target: https://github.com/psf/black
+.. |changelog-badge| image:: https://img.shields.io/badge/-change%20log-purple
+   :alt: Change log
+   :target: https://github.com/akaihola/darker/blob/master/CHANGES.rst
+
+What is Darker?
+===============
+
+.. include:: ../README.rst
+   :start-after: What?
+   :end-before: Why?
```

Comment:
Could we copy the titles as well, like:
```suggestion
.. include:: ../README.rst
   :start-at: What?
   :end-before: Why?
```

Reply:
Or is it that we don't get proper references and tables of content if the headings are not actually literally present in the Sphinx document?

Reply:
Right - we don't have anything to reference to doing that